test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
@Gambit142 all changes included, I just have one question when it comes to the test file, I upload this from my computer, in this case the gitignore file would not work? I am not using the bash, should I just delete it from Github?

Reply:
hi @edgardsoza you can decide to delete it from GitHub, then on the repository on the local machine, you would `git pull` before making changes to avoid merge conflicts. So the steps should be:

- Delete file from GitHub
- `git pull` changes before making changes to the local repository
- make changes, 
- `git add` and `git commit` 
- `git push`